modules/storage-api/src/testFixtures/java/org/apache/ignite/internal/storage/BaseMvStoragesTest.java

```diff
@@ -181,7 +159,7 @@ protected static List<IgniteBiTuple<TestKey, TestValue>> drainToList(Cursor<Read
         }
     }
 
-    protected final void assertRowMatches(TableRow rowUnderQuestion, TableRow expectedRow) {
+    protected final void assertRowMatches(@Nullable TableRow rowUnderQuestion, TableRow expectedRow) {
```

Comment:
Why is **rowUnderQuestion** `@Nullable`? Logically, it just can't be.

Reply:
First of all, read returns `nullable`, so we pass nullable to the non-null, bad.
Second, in the first line of method we assert it not to be null. So it is nullable

Reply:
Ok